app/style/video-calling.less

```diff
@@ -41,7 +41,7 @@
 .video-calling {
   &--active {
     position: absolute;
-    z-index: 101;
+    z-index: 1001;
```

Comment:
We should not use these hard codes values here. We have z-index variables defined to avoid exactly this problem. Please use them.